lib/modules/coverage/source_map.js

```diff
@@ -1,29 +1,64 @@
+class EmptySourceMap {
```

Comment:
Do you need a class if you only have static function in it?
Would it not be better in terms of intention to have something like this:
```
  const EmptySourceMap = {
    createRelativeTo: function....
  }
```

I know this is the same just that behind a word `class` I always expect instance variable